CONTRIBUTING.md

```diff
@@ -112,6 +112,13 @@ You need install some dependencies before building a binary.
   $ export PKG_CONFIG_PATH="/usr/lib/pkgconfig"
   ```
 
+#### openSUSE
+
+```shell
+$ sudo zypper -n in libgpgme-devel libseccomp-devel systemd-devel
+$ export PKG_CONFIG_PATH="/usr/lib/pkgconfig"
```

Comment:
is the export path actually correct?
I have no idea why this was added but it looks wrong, at least on fedora this path does not exists. And I really do not understand why this would be needed as the distro defaults should just work out of the box.

Reply:
Let me check if this is strictly necessary.

Reply:
It's not necessary on Fedora 39 and openSUSE Tumbleweed to run `make` and `make binaries`. I think we can remove it.

Reply:
Also Debian doesn't require `export PKG_CONFIG_PATH="/usr/lib/pkgconfig"`. On all systems I got a `bin/podman` binary where at least `podman --version` works.

Reply:
Let's remove it then as it is misleading, tbh I am confused why this section exists in the first place. These are not the only build dependencies (https://podman.io/docs/installation#build-and-run-dependencies) which should be much more accurate and is already linked earlier in the document.
Thus I think this entire section can be removed.

Reply:
When you refer to the section, you mean the `Prerequisite before build` section?

Reply:
yes

Reply:
I would keep this as it is, and tidy the build instructions in another MR. I have the feeling that is going to be another discussion.

Reply:
https://github.com/containers/podman/pull/22468